holidays/countries/syrian_arab_republic.py

```diff
@@ -0,0 +1,214 @@
+    GREGORIAN_CALENDAR,
+    JAN,
+    FEB,
+    MAR,
+    APR,
+    MAY,
+    JUN,
+    JUL,
+    AUG,
+    SEP,
+    OCT,
+    NOV,
+)
+from holidays.calendars.julian import JULIAN_CALENDAR
+from holidays.constants import GOVERNMENT, OPTIONAL, PUBLIC
+from holidays.groups import ChristianHolidays, InternationalHolidays, IslamicHolidays
+from holidays.holiday_base import HolidayBase
+
+
+class SyrianArabRepublic(HolidayBase, ChristianHolidays, InternationalHolidays, IslamicHolidays):
+    """
+    Syrian Arab Republic holidays.
+
+    References:
+    * [Timeanddate.com](https://www.timeanddate.com/holidays/syria/2025)
+    * [OfficeHolidays.com](https://www.officeholidays.com/countries/syria/2025)
+    * [PublicHolidays.me](https://publicholidays.me/syria/2025-dates/)
+    * [Wikipedia](https://en.wikipedia.org/wiki/Public_holidays_in_Syria)
+
+    Holidays are regulated by the Syrian government and may vary each year.
+    """
+
+    country = "SY"
+    default_language = "ar"
+    # %s (estimated).
+    estimated_label = tr("(تقدير) %s")
+    supported_categories = (GOVERNMENT, OPTIONAL, PUBLIC)
+    supported_languages = ("ar", "en_US")
+    # Syria gained independence on April 17th, 1946.
+    start_year = 1947
+
+    def __init__(self, islamic_show_estimated: bool = True, *args, **kwargs):
+        """
+        Args:
+            islamic_show_estimated:
+                Whether to add "estimated" label to Islamic holidays name
+                if holiday date is estimated.
+        """
+        ChristianHolidays.__init__(self, JULIAN_CALENDAR)
+        InternationalHolidays.__init__(self)
+        IslamicHolidays.__init__(
+            self, cls=SyrianArabRepublicIslamicHolidays, show_estimated=islamic_show_estimated
+        )
+        super().__init__(*args, **kwargs)
+
+    def _populate_public_holidays(self):
+        # New Year's Day.
+        self._add_new_years_day(tr("رأس السنة الميلادية"))
+
+        # Revolution Day.
+        if self._year >= 1963:
+            self._add_holiday_mar_8(tr("عيد الثورة"))
+
+        # Mother's Day.
+        if self._year >= 1950:
+            self._add_holiday_mar_21(tr("عيد الأم"))
+
+        # Independence Day.
+        if self._year >= 1946:
+            self._add_holiday_apr_17(tr("عيد الجلاء"))
+
+        # Labor Day.
+        if self._year >= 1948:
+            self._add_labor_day(tr("عيد العمال"))
+
+        # Martyrs' Day.
+        if self._year >= 1947:
+            self._add_holiday_may_6(tr("عيد الشهداء"))
+
+        # Anniversary of the October Liberation War.
+        if self._year >= 1974:
+            self._add_holiday_oct_6(tr("عيد حرب تشرين التحريرية"))
+
+        # Christmas Day.
+        self._add_christmas_day()
+        if self._year >= 2024:
+            self._add_christmas_day(tr("عيد الميلاد المجيد"), GREGORIAN_CALENDAR)
+
+        # Boxing Day.
+        if self._year >= 2024:
+            self._add_christmas_day_two(tr("يوم الصناديق"))
```

Comment:
No need to keep things separate since both shares the same condition
```suggestion
        if self._year >= 2024:
            # Christmas Day.
            self._add_christmas_day(tr("عيد الميلاد المجيد"), GREGORIAN_CALENDAR)

            # Boxing Day.
            self._add_christmas_day_two(tr("يوم الصناديق"))
```